web-client/src/pages/admin/AdminInventoryPage.tsx

```diff
@@ -1,5 +1,100 @@
-import UnderConstruction from "@/components/UnderConstruction";
+import { Button } from "@/components/ui/button";
+import { Search } from "lucide-react";
+import { ChevronDown } from "lucide-react";
+import { DropdownMenu, DropdownMenuItem, DropdownMenuTrigger, DropdownMenuContent } from "@/components/ui/dropdown-menu";
+import InventoryTable from '@/components/InventoryTable';
+import { useNavigate } from 'react-router-dom';
+import { useGetSocks } from '@/api/inventory/queries';
+import { useState } from 'react';
 
 export default function AdminInventoryPage() {
-  return <UnderConstruction pageName="Inventory" />;
+  const navigate = useNavigate();
+
+  const [page, setPage] = useState(1);
+  const limit = 8;
```

Comment:
Extract the number into a constant in this same file at the top like: `SOCKS_RESULTS_LIMIT`

Reply:
Changes in this commit:
https://github.com/sockify/sockify/pull/120/commits/5a0bcfa392ffe664154ec5e6f15c65839fa9a85a
Added OCKS_RESULTS_LIMIT constant. And replaced all instances of limit with OCKS_RESULTS_LIMIT